docs/components/tables/index.html

```diff
@@ -10,6 +10,17 @@ <h2 class="hxSectionTitle" id="default-table">Default Table</h2>
     <em>a.k.a. "Unbound Table"</em>
   </p>
 
+  <dl>
+    <dt><code>.hxTable</code></dt>
```

Comment:
Did we just take out code blocks on these because they're bolded already in style?

Reply:
good point! these differ a little bit -- they are css classes that can be copy/pasted, the others were for attributes/properties